base/bucket.go

```diff
@@ -205,44 +204,30 @@ func (b BucketSpec) GetViewQueryTimeoutMs() uint64 {
 }
 
 func (b BucketSpec) TLSConfig() *tls.Config {
-	tlsConfig, err := TLSConfigForX509(b.Certpath, b.Keypath, b.CACertPath)
+	tlsConfig, err := TLSConfigForX509(b.CACertUnsetTlsSkipVerify, b.CACertPath, b.Certpath, b.Keypath)
```

Comment:
I don't see TLSConfigForX509 being used anywhere else.  Given that, can we just move it inside this method, since otherwise this method is just doing a bit of error handling?